bedwars-plugin/src/main/java/com/tomkeuper/bedwars/addon/AddonManager.java

```diff
@@ -124,6 +136,7 @@ public void loadAddons() {
     public void registerAddon(Addon addon){
         if (addon == null) return;
         if (registeredAddons.contains(addon)) return;
+        if (registeredAddons.stream().map(Addon::getPlugin).collect(Collectors.toList()).contains(addon.getPlugin())) return;
```

Comment:
What is the point of checking this way?

Reply:
Preventing the addon to be added multiple times on the list since when creating a new instance of an extended addon class it wont be the same as the old one

Reply:
Fair point. Looks good!